### PR TITLE
Problem: Copyright notice is very confusing

### PR DESCRIPTION
**Pull request by gazhayes**
It doesn't show in plain text, but when the MD is displayed it shows the whole copyright notice on one single line and it doesn't make sense.
Solution: add line break to copyright notice and fix grammar.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Monero
 
-Copyright (c) 2014-2017, The Monero Project
-Portions Copyright (c) 2012-2013, The Cryptonote developers
+Copyright (c) 2014-2017 The Monero Project.
```
Comment (MaxXor) [inline README.md]: These added spaces are not needed for a line break.

**Comment by MaxXor** [comment]
@erikd I checked it and it seems the whitespace is needed to force a new line.
Otherwise I suggest the following edit with a blank line:
>Copyright (c) 2014-2017 The Monero Project.
>
>Portions Copyright (c) 2012-2013 The Cryptonote developers.

**Comment by moneromooo-monero** [comment]
Not sure why we even have markup in a README in the first place.

File: README.md (hunk 1 above)
Comment (erikd) [inline README.md]: There is trailing whitespace there on that line.

**Comment by gazhayes** [comment]
The trailing whitespace is just Github markdown for new line.

**Comment by erikd** [comment]
No,please look at the diff. The line with "The Monero Project." has trailing whitespace and the other one does not.

**Comment by fluffypony** [comment]
@erikd that's because you only need to force a linebreak on the first line, the double-enter after the second line is a paragraph break:)

**Comment by fluffypony** [comment]
@gazhayes congrats on your first merged pull request!!

thumbs up

**Pull request merged**